prime.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/ruby/prime"
   spec.licenses      = ["Ruby", "BSD-2-Clause"]
 
-  spec.files         = [".gitignore", "Gemfile", "LICENSE.txt", "README.md", "Rakefile", "bin/console", "bin/setup", "lib/prime.rb", "prime.gemspec"]
+  spec.files         = [".gitignore", "Gemfile", "LICENSE.txt", "README.md", "Rakefile", "lib/prime.rb", "prime.gemspec"]
```

Comment:
You can remove the git ignore and the Gemfile, too, they do nothing. And the gemspec.

Reply:
I removed them at https://github.com/ruby/prime/commit/586cbdd2af7a5905121c2c93555cdc0bc3c07b50